next.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
       process.env.HUMAN_INTERNET_URL || "https://human-internet.org/",
     DYNAMIC_LINK_DOMAIN:
       process.env.DYNAMIC_LINK_DOMAIN || "https://devme.bettersocial.org",
+    WIX_URL: process.env.WIX_URL || 'https://home.helio.social/'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `WIX_URL` environment variable with a default value is a good practice. However, it's important to ensure that this new environment variable is documented and added to any deployment scripts or instructions so that it can be properly configured in different environments.
```diff
+    WIX_URL: process.env.WIX_URL || 'https://home.helio.social/'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->